code/modules/modular_computers/computers/subtypes/preset_console.dm

```diff
@@ -176,15 +176,6 @@
 	ai_slot = new/obj/item/computer_hardware/ai_slot(src)
 	card_slot = new/obj/item/computer_hardware/card_slot(src)
 
-/obj/item/modular_computer/console/preset/merchant/nka
-	_app_preset_type = /datum/modular_computer_app_presets/merchant/nka
-
-/obj/item/modular_computer/console/preset/merchant/guild
-	_app_preset_type = /datum/modular_computer_app_presets/merchant/guild
-
-/obj/item/modular_computer/console/preset/merchant/golden_deep
-	_app_preset_type = /datum/modular_computer_app_presets/merchant/golden_deep
```

Comment:
are these modular computer app presets actually deleted? are they even used?

Reply:
Thanks for the catch, no they are no longer used as they no longer have a purpose